package-lock.json

```diff
@@ -2327,6 +2332,16 @@
       "dev": true,
       "license": "MIT"
     },
+    "node_modules/@types/amqplib": {
+      "version": "0.8.2",
+      "resolved": "https://registry.npmjs.org/@types/amqplib/-/amqplib-0.8.2.tgz",
+      "integrity": "sha512-p+TFLzo52f8UanB+Nq6gyUi65yecAcRY3nYowU6MPGFtaJvEDxcnFWrxssSTkF+ts1W3zyQDvgVICLQem5WxRA==",
+      "dev": true,
+      "dependencies": {
+        "@types/bluebird": "*",
+        "@types/node": "*"
+      }
+    },
```

Comment:
Currently, `package-lock.json` contain a lot of redundant statements. Please revert changes